GameDefinition/Content/ItemKindActor.hs

```diff
@@ -474,6 +474,24 @@ alligator = ItemKind
                , ("eye 6", COrgan), ("ear 9", COrgan)
                , ("animal brain", COrgan) ]
   }
+octopus = ItemKind
+  { isymbol  = 'o'
+  , iname    = "Giant octopus"
+  , ifreq    = [ ("animal", 100), ("mobile", 1), ("mobile animal", 100) ]
+  , iflavour = zipPlain [BrMagenta]
+  , icount   = 1
+  , irarity  = [(1, 5)]
+  , iverbHit = "thud"
+  , iweight  = 15000
+  , idamage  = 0
+  , iaspects = [ AddSkill SkMaxHP 14, AddSkill SkMaxCalm 80
```

Comment:
Perfect. And I'd add this:

https://github.com/AllureOfTheStars/Allure/blob/74ecdf787d3cbffca8a80c8ac6f5d20f187c941f/GameDefinition/Content/ItemKindActor.hs#L468